tests/utils.go

```diff
@@ -1937,16 +1946,29 @@ func NewRandomFedoraVMIWithDmidecode() *v1.VirtualMachineInstance {
 }
 
 func GetGuestAgentUserData() string {
+	virtClient, err := kubecli.GetKubevirtClient()
+	PanicOnError(err)
+
+	dnsServerIP, err := getClusterDnsServiceIP(virtClient)
+	PanicOnError(err)
+
+	ipv6UserDataString := ""
+	if netutils.IsIPv6String(dnsServerIP) {
+		ipv6UserDataString = fmt.Sprintf(`sudo ip -6 addr add fd2e:f1fe:9490:a8ff::2/120 dev eth0
```

Comment:
why do we have to add the ip in cloud init? is the ip not automatically provided to the VMI via dhcp?

Reply:
We don't have dhcp server for ipv6. Vm ipv6 addresses should be configured manually (via cloud-init, console, etc.) Please refer to the PR description for more details.